includes/class-amp-story-legacy-post-type.php

```diff
@@ -1115,244 +1115,6 @@ public static function add_custom_stories_styles() {
 		self::enqueue_general_styles();
 	}
 
-	/**
```

Comment:
Just to confirm: the functionality of the legacy Stories editor didn't change with this change, correct? Looks like not, just ensuring that we're keeping the legacy editor as it was before.

Reply:
Just moved some logic around to make it share about. Should be no breaking changes to legacy CPT.